### ISSUE TITLE
Does vim-hoogle work with stack?

### ISSUE THREAD

**Issue by timchu** [Twinside/vim-hoogle#7]
I'm using the stack package manager, and vim-hoogle doesn't seem to just work the way it does in  cabal. Whenever I hoogle something in vim, I get the error

'''
"HoogleSearch" [Not edited] --No lines in buffer--
shell returned 127
'''

**Comment by Twinside** [comment]
vim-hoogle work with `hoogle` (the executable which should be in your path), it doesn't give a fuck about `cabal` or `stack`

**Comment by will-ob** [comment]
@timchu did you ever find a fix for this? I see something similar with a missing db error

```
Error, database does not exist (run 'hoogle generate' first)
    Filename: /home/will/.hoogle/default-haskell-5.0.1.hoo
```

The ~/.hoogle directory does not appear to exist at all. Even after running `hoogle data`.
`hoogle` on my command line works fine

**Comment by lazamar** [comment]
Yes, it can be used with stack by adding this line to your `.vimrc`

``` vim
let g:hoogle_search_bin = 'stack hoogle --'
```
